example/main.go

```diff
@@ -1,16 +1,18 @@
 package main
+
 // Copyright © 2015 Ian Denhardt <ian@zenhack.net>
 // Licensed under the GPLv3 or later.
 // See COPYING at the root of the repository for details.
 
 import (
-	".."
 	"flag"
 	"fmt"
+
+	"github.com/kalbasit/go.notmuch"
```

Comment:
relative imports are not allowed in Go. but yes this should be your fork (I'll fix it, also trying to fix Travis now).